src/locale/locales/hu/messages.po

```diff
@@ -567,7 +567,7 @@ msgstr "Az alapértelmezett hírfolyamban az általad követett személyek bejeg
 
 #: src/screens/Settings/components/ChangeHandleDialog.tsx:389
 msgid "Add the following DNS record to your domain:"
-msgstr "Vedd fel az alábbi DNS-rekordot a tartományodhoz:"
+msgstr "Add hozzá az alábbi DNS-rekordot a domainedhez:"
```

Comment:
‘Add hozzá’ means the same thing but sounds more English than Hungarian – I don't think it needs to be modified.
‘Tartomány’ is the Hungarian term for domain – please see [Wikipedia](https://hu.wikipedia.org/wiki/Tartom%C3%A1nyn%C3%A9v) or [Microsoft Terminology Search](https://msit.powerbi.com/view?r=eyJrIjoiODJmYjU4Y2YtM2M0ZC00YzYxLWE1YTktNzFjYmYxNTAxNjQ0IiwidCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsImMiOjV9).

Reply:
Yeah, probably the first part could stay as is, but I never heard people referring to a domain as "tartomány".

Reply:
I understand if you don't really use that term in person (I myself end up using quite a lot of English terms as-is at work, since usually they're shorter), but ‘tartomány’ has a long precedent and is generally used in software.